test/gtest/solver_convasm3x3u.cpp

```diff
@@ -0,0 +1,90 @@
+/*******************************************************************************
+ *
+ * MIT License
+ *
+ * Copyright (c) 2022 Advanced Micro Devices, Inc.
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in all
+ * copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ *
+ *******************************************************************************/
+#include "solver.hpp"
+
+struct ConvFwdSolverTestFloat : ConvFwdSolverTest<float>
+{
+};
+
+template <typename Solver>
+void SolverFwd(const miopen::TensorDescriptor& inputDesc,
+               ConstData_t input,
+               const miopen::TensorDescriptor& wDesc,
+               ConstData_t weight,
+               const miopen::TensorDescriptor& outputDesc,
+               Data_t output,
+               const miopen::ConvolutionDescriptor& convDesc,
+               const ConvTestCase& conv_config,
+               bool& test_skipped)
+{
+    auto&& handle = get_handle();
+
+    Solver solv{};
+
+    const auto tensors =
+        miopen::ConvFwdTensors{inputDesc, input, wDesc, weight, outputDesc, output};
+
+    auto ctx = miopen::ConvolutionContext{
+        inputDesc, wDesc, outputDesc, convDesc, miopen::conv::Direction::Forward};
+
+    ctx.SetStream(&handle);
+    ctx.DetectRocm();
+
+    if(!solv.IsApplicable(ctx))
+    {
+        test_skipped = true;
+        GTEST_FAIL() << solv.SolverDbId() << "ConvAsm3x3U Not Applicable for this problem"
```

Comment:
```suggestion
        GTEST_SKIP() << solv.SolverDbId() << "ConvAsm3x3U Not Applicable for this problem"
```

Reply:
@JehandadKhan replaced with GTEST_SKIP()

Reply:
Good. This change resolves https://github.com/ROCmSoftwarePlatform/MIOpen/pull/1877#issuecomment-1315461491.